pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
```

Comment:
this is missing comments for public function https://blog.golang.org/godoc-documenting-go-code

Reply:
Fixed.